examples/globus_ls.py

```diff
@@ -0,0 +1,42 @@
+"""A simple script that does a 'globus ls' using lta's GlobusTransfer."""
+
+import logging
+from pathlib import Path
+import json
+import asyncio
+from lta.transfer.globus import GlobusTransfer
+
+
+def globus_ls_recursive(gt: GlobusTransfer, collection: str, path: Path):
+    """Like 'globus ls <collection>' but recursively for every subdir."""
+
+    print(f"\n\nglobus ls {path}")
+
+    entries = gt._transfer_client.operation_ls(collection, path=str(path))["DATA"]
+
+    for e in entries:
+        fullpath = path / e["name"]
+        print(f"\n-> {fullpath}")
+        print(json.dumps(e, indent=4))
```

Comment:
This produces a JSON per directory in the recursive tree. Is it worth collecting up into a single JSON blob (maybe on a `json: bool` parameter)?

Then again, it is an example, so maybe simple clarity it better than every bell and whistle in the world. :thinking:

Reply:
I'll keep it for that reason, since it's a debugging/test script.